cloud_build/scripts/stage_site_and_comment_on_github.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+deploy_to_firebase_staging_channel () {
+    echo "Deploying website to a staging channel on firebase..."
+
+    FIREBASE_DEPLOY_RESPONSE=$(firebase hosting:channel:deploy --expires 7d pr$PR_NUMBER-$HEAD_BRANCH --project=$PROJECT_ID)
+    echo "$FIREBASE_DEPLOY_RESPONSE"
+    FIREBASE_STAGING_URL=$(grep -Eo "https://$PROJECT_ID--[a-zA-Z0-9./?=_%:-]*" <<< "$FIREBASE_DEPLOY_RESPONSE")
```

Comment:
This is much easier to understand for me, thank you!

Could you add a comment above the line explaining what it's doing though. The grep and regex might confuse others in the future.

Reply:
Added!